plugins/mangatx/info.json

```diff
@@ -0,0 +1,8 @@
+{
+	"id": "Mangatx",
+	"title": "Mangatx",
+	"author": "Robzimpulse - kugelfang.killaruna@gmail.com",
+	"version": "v1.0",
+	"placeholder": "Mangatx Gallery URL",
```

Comment:
Perhaps it would be clearer to use "Mangatx manga URL" instead?

Reply:
sure, updated on commit 73ce376